### DISCUSSION TITLE
Model number

### DISCUSSION THREAD

**Discussion by tschmidtb51** [netbox-community/netbox#12969] [Help Wanted!]
I looked into the device types and saw the field `model name`. However, e.g. for Siemens, the `model number` (6ES7222-1BD30-0XB0) is different from the `Model Name` (Signal Board SB1222, 4 DQ 24VDC 200KHz). Similar example is Lenovo Thinkpad T470p (`Model Name`), Type: 20J6-003DGE. Is the model number field missing or how should that be modeled?

**Comment by markkuleinio** [comment]
There is the part number field:



Manufacturer | Lenovo
-- | --
Model Name | ThinkPad X1 Carbon Gen 10
Part Number | 21CBCTO1WW
